src/coreclr/vm/method.cpp

```diff
@@ -2626,15 +2626,22 @@ MethodDesc* MethodDesc::GetMethodDescFromStubAddr(PCODE addr, BOOL fSpeculative
 
     MethodDesc *  pMD = NULL;
 
-    // Otherwise this must be some kind of precode
-    //
-    PTR_Precode pPrecode = Precode::GetPrecodeFromEntryPoint(addr, fSpeculative);
-    PREFIX_ASSUME(fSpeculative || (pPrecode != NULL));
-    if (pPrecode != NULL)
+    EX_TRY
```

Comment:
This should be DAC-only change. Catching the exceptions at runtime is unnecessary - perf overhead, hiding bugs.

Reply:
(The DAC-only try/catch can be either here or in request.cpp. I am not sure what's better.)

Reply:
I'm moving it to request.cpp.